lib/configService.js

```diff
@@ -55,7 +56,21 @@ function processEnvironmentVariables() {
             'IOTA_AMQP_RETRY_TIME',
             'IOTA_HTTP_HOST',
             'IOTA_HTTP_PORT',
-            'IOTA_HTTP_TIMEOUT'
+            'IOTA_HTTP_TIMEOUT',
+            'IOTA_CB_URL',
```

Comment:
I'm not fully sure, but I'd say that all these env var are the ones that are actually managed by IOTA Node Lib (for instance, see https://github.com/telefonicaid/iotagent-node-lib/blob/master/lib/commonConfig.js#L115). IOTA specific code shouldn't care about them.

Reply:
I think you are right so should I remove the changes done in this file ?

Reply:
Yes, I think so.

Reply:
Fixed with https://github.com/telefonicaid/iotagent-json/pull/380/commits/f78515c77f7455bbde2fce8b806b85c99ecda014